### PR TITLE
Can you please help us review our game logic.. Thanks

### PR DESCRIPTION
**Pull request by rootDEV2990**
Please help us review our games logic branch. Game will ask for a rematch if winner is found or game results in a cats game. If rematch is not accepted game stops. 

Thank you. 🙏🏽

### CODE REVIEW

**Comment by rootDEV2990** [comment]
Screen Shot 2020-06-29 at 11 38 20 AM
Why does stickler say theres errors if error list is empty??

**Comment by uryelah** [comment]
Hi **Miguel Enciso Sanchez & Joe Abraham**, I'm Sarah your reviewer for this PR 👩‍🚀 

About the rubocop error, it could be an issue with the .rubocop.yml file. 
Try it again with the same [config found in this one.](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)
Or even better,[ follow the instruction here to use github actions](https://github.com/microverseinc/linters-config/tree/master/ruby) instead of stickler.

You are just missing adding the testing info to your readme file, then you'll be ready for bigger and better projects! 💪 

# Required Changes 🏗️ 🐛 💡

- [x] Good job with the classes! But the only file to contain 'puts' should be the main.rb.
Refactor your class methods to return the strings instead of the put statements. It's possible to return strings with multiple lines in ruby, make a quick google search to figure out how exactly.

- [x] Please rewrite your PR message, checking the grammar, and making sure there are no typos around. I recommend using the Grammarly browser extension for that.


-----

 **Kindly fix the pointed out issues before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by rootDEV2990** [comment]
Updated code with coding partner for new review.

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  def initialize
+    @board = Array.new(9, ' ')
+  end
+
+  def game_board
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+    puts '-----------'
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+    puts '-----------'
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
```
Comment (mosesogwo) [inline lib/board.rb]: puts and prints are only allowed from main.rb.
You can return your game board as a string value which you then print out from main.rb
Go back and see the previous review on this point.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**:clap:Hello!, please attend to the issue raised in the previous review before submitting another code review request.**

_**Good luck :stars:**_

### Status: Changes required :x:

File: lib/board.rb (hunk 1 above)
Comment (rootDEV2990) [reply lib/board.rb]: OK we made some changes to the game_board method and removed puts from the board.rb file. Thanks.

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  def game_board
+    play_board = [
+    " #{@board[0]} | #{@board[1]} | #{@board[2]} ",
+    "-----------",
+    " #{@board[3]} | #{@board[4]} | #{@board[5]} ",
+    "-----------",
+    " #{@board[6]} | #{@board[7]} | #{@board[8]} "
+    ]
+    puts play_board.join("\n") + "\n"
```
Comment (kobiyoyo) [inline lib/board.rb]: No `puts/print` and `get.chomp` outside `main.rb` file.

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team      
#### Great Job so far, but there are a few more things required. 
### General
- [x]  Check the comment below
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by rootDEV2990** [comment]
Thanks we forgot to push last change I think, sorry about that but we took it out of the board.rb file. Can you help us see if its ok now? Thank you. 
Screen Shot 2020-07-01 at 4 29 09 PM

**Review by davidmrtz-dev — APPROVED** [review]
****
## Hi. 😄 
****
Great Job 💯 👍  @rootDEV2990 , You correctly implemented the required changes.
You can move onto the next project. 🎊 
Happy coding. 👨‍🚀  
****
### Status: Approved. ✅
****

**[TSE:  David Elí]** 👨‍💻 
Slack: @ David Eli Martinez Garcia

**Pull request merged**